setup.py

```diff
@@ -18,8 +18,18 @@
     name="pymatgen-analysis-alloys",
     packages=find_namespace_packages(include=["pymatgen.analysis.*"]),
     version="0.0.6",
-    install_requires=["pymatgen>=2023.7.17", "shapely>=1.8.2"],
+    install_requires=[
+        "monty",
+        "networkx",
+        "numpy",
+        "pandas",
+        "plotly",
+        "pymatgen>=2023.7.17",
+        "scipy>=1.8.0",
+        "shapely>=1.8.2",
+        ],
     extras_require={},
+    setup_requires=["setuptools>=64"],
```

Comment:
I might keep this line of change in this PR as well as setuptools < 58.3.0 is known to have [buggy behaviour with package data handling](https://github.com/abravalheri/experiment-setuptools-package-data)